rllib/tuned_examples/bc/cartpole_recording.py

```diff
@@ -45,8 +45,9 @@
     )
     .offline_data(
         output="local:///tmp/cartpole/",
-        output_write_episodes=True,
+        output_write_episodes=False,
         output_max_rows_per_file=1000,
+        output_compress_columns=["obs", "new_obs", "actions"],
```

Comment:
Can we add a one-line comment above to explain a bit?

Reply:
Will do!